src/ucs/debug/memtrack.c

```diff
@@ -512,3 +512,13 @@ void ucs_memtrack_releasing(void *ptr)
 {
     ucs_memtrack_releasing_internal(ptr);
 }
+
+ucs_status_t ucs_alloc_path_buffer(char **buffer, const char *name)
+{
+    *buffer = ucs_malloc(PATH_MAX, name);
+    if (*buffer == NULL) {
```

Comment:
Traditionally, allocators don't assign the "inout" ptr, unless the allocation succeeds.

Reply:
done